internal/output/github/github.go

```diff
@@ -0,0 +1,68 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+// Package github implements interface to GitHub API.
+package github
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/google/go-github/v32/github"
+	"golang.org/x/oauth2"
+)
+
+// Output implements interface to GitHub API.
+type Output struct {
+	client *github.Client
+}
+
+// NewOutput creates new GitHub API output.
+func NewOutput() *Output {
+	output := &Output{}
+
+	token, exists := os.LookupEnv("GITHUB_API_TOKEN")
+	if !exists {
+		fmt.Println("GITHUB_API_TOKEN is missing, GitHub API integration is disabled")
```

Comment:
In the bot and in conform we use `GITHUB_TOKEN`. This env var is also respected by `hub` and the `gh` CLIs. Super nit, but I think maybe we should use that instead? Then my token that I have exported in my shell work, and it aligns with the official GH way.

Reply:
updated, thanks!